cmd/influxd/restore/restore.go

```diff
@@ -157,7 +157,6 @@ func (cmd *Command) unpackMeta() error {
 	i += int(length)
 
 	// Size of the node.json bytes
-	length = int(binary.BigEndian.Uint64(b[i : i+8]))
 	i += 8
 	nodeBytes := b[i:]
```

Comment:
We should actually use `length` here instead of assuming the size of remaining data is correct, i.e. 

```go
nodeBytes := b[i : i+length]
```